r2_laser_scan_filter:
  scan_filter_chain:
  - name: bounds_repeat
    type: sd_sensor/LaserScanAngularBoundsListFilter
    params:
      angles: []
      angles2: []

r2_laser_pattern_detector:
  detector_frequency: 5.0

  # ? = any, 0 = below threshold, 1 = greater than threshold, X = obstructed
  # 
  # Degree              0         60        120       180        240       300       
  isTirette_pattern:   "XXX??????????????????????????????????????????????????????XXX"
  isBackFree_pattern:  "??111???????????????????????????????????????????????????????"
  isLeftFree_pattern:  "??????????????????????????11????????????????????????????????"
  isFrontFree_pattern: "??????????????????????????????11????????????????????????????"
  isRightFree_pattern: "??????????????????????????????????11????????????????????????"
  
  # Distance threshold in meters
  isTirette_threshold: 0.1
  isLeftFree_threshold: 0.6
  isFrontFree_threshold: 0.3
  isRightFree_threshold: 0.6
  isBackFree_threshold: 0.3

mpu6050:
  frequency: 50
  frame: 'r2/imu_link'
  frequency: 50
  update_bias_at_startup: true
  update_bias_at_startup_count: 100